packages/block-library/src/breadcrumbs/index.php

```diff
@@ -100,9 +100,25 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 			$show_terms = $attributes['prefersTaxonomy'];
 		}
 
-		// Build breadcrumb trail.
+		// Add post type archive link if applicable.
+		$post_type_object = get_post_type_object( $post_type );
+		$archive_link     = get_post_type_archive_link( $post_type );
+		if ( $archive_link && untrailingslashit( home_url() ) !== untrailingslashit( $archive_link ) ) {
+			$label = $post_type_object->labels->archives;
+			if ( 'post' === $post_type ) {
+				$page_for_posts = get_option( 'page_for_posts' );
+				if ( $page_for_posts ) {
+					$label = block_core_breadcrumbs_get_post_title( $page_for_posts );
+				}
+			}
```

Comment:
How do you feel about this:

```suggestion
			if ( 'post' === $post_type && $page_for_posts = get_option( 'page_for_posts' ) ) {
				$label = block_core_breadcrumbs_get_post_title( $page_for_posts );
			}
```

Reply:
Ah, interesting. I used to love the brevity of the `if ( $var = something() )` construct, I assume linter's not happy because we're both assigning and checking at the same time (and it might look like an error - `=` vs `===`)